sample.json

```diff
@@ -0,0 +1,15 @@
+{
+  "uuid" : "path/to/example.tif",
+  "projection" : "PROJCS["NAD83 / Massachusetts Mainland", ...",
+  "bbox" : [-180,-90,180,90],
+  "footprint" : "POLYGON((-180 -90, -180 90, 180 90, 180 -90, -180 -90))",
+  "gsd" : 0.35,
+  "file_size" : 1024,
+  "acquisition_start" : "2015-05-03T13:00:00.000",
+  "acquisition_end" : "2015-05-04T13:00:00.000",
+  "title" : "An scene from some satellite",
```

Comment:
@lossyrob By per-dataset you mean per-bucket? Or a dataset within a bucket? 

I think even if we're having duplicative information, having complete metadata per image will be an advantage going forward. It seems that it will be useful to have all information about the image in the json file. This makes it dead simple for anyone to read a json file and know information about the image. I think it'd be fine to have a per-bucket json file of information.